server/models.py

```diff
@@ -499,6 +500,28 @@ class RoundContestResult(BaseModel):
     result = Column(Integer, nullable=False)
 
 
+class JurisdictionResult(BaseModel):
+    round_id = Column(
+        String(200), ForeignKey("round.id", ondelete="cascade"), nullable=False
+    )
+    contest_id = Column(
+        String(200), ForeignKey("contest.id", ondelete="cascade"), nullable=False,
+    )
+    jurisdiction_id = Column(
+        String(200), ForeignKey("jurisdiction.id", ondelete="cascade"), nullable=False,
+    )
+    contest_choice_id = Column(
+        String(200),
+        ForeignKey("contest_choice.id", ondelete="cascade"),
+        nullable=False,
+    )
+    result = Column(Integer, nullable=False)
+
+    __table_args__ = (
+        PrimaryKeyConstraint("round_id", "jurisdiction_id", "contest_choice_id"),
```

Comment:
@benadida do you think it's worth having `contest_id` in here as well? I noticed that you didn't include it in the primary key for `RoundContestResult`, so was following that pattern, but part of me feels like it would be clearer to say that it is part of the primary key

Reply:
I think you can go either way, but if you include in the primary key some keys that effectively have redundant info, I think it's important to have those keys foreign-reference the table that ensures they are internally consistent.

Reply:
Ok makes sense. I'll prob leave it out.

But your comment made me think: is it worth adding a constraint on `contest_id` and `jurisdiction_id` to make sure they're related via the `contest_jurisdiction` table?